node/network/bridge/src/lib.rs

```diff
@@ -169,22 +169,55 @@ where
 		let action = {
 			let subsystem_next = ctx.recv().fuse();
 			let mut net_event_next = event_stream.next().fuse();
+			let mut req_res_event_next = bridge.request_multiplexer.next().fuse();
 			futures::pin_mut!(subsystem_next);
 
 			futures::select! {
 				subsystem_msg = subsystem_next => Action::from(subsystem_msg),
 				net_event = net_event_next => Action::from(net_event),
+				req_res_event = req_res_event_next  => Action::from(req_res_event),
 			}
 		};
 
 		match action {
 			Action::Nop => {}
-			Action::Abort => return Ok(()),
+			Action::Abort(reason) => match reason {
+				AbortReason::SubsystemError(err) => {
```

Comment:
If `AbortReason` becomes an `Error` type, this could become trivial a well.

Reply:
hvhggurgvlefhurehgygkiuregueriglheioygruivklhfbhugnjjhrgguiruyeruhhgjfrhegfefuyhgj,fhuwuhryryuryrgeurkyergkehrgfkwrgkegyrkeyuiwkgrygrwuedygdfguefgufuirgeyftuyfgrguygtfugycasidqweikgdewikegrfiwukqgfcygsdygdwyegyeiqgyeuduysgagyuawduyqwgeyyuqgweywqguedgwewrggggggghhhhhkkgdciushfjusfgdxposdhdfp;ewedcsdkiaiwoiqqwopijdopwujdeoew9uoijkioewowedodwepi;oqwejoeujeiowufewiudkewujiuewoeuwqeuirwqjrfiewyhjcswduanwshewanjdsdnwsauehwuyeduwyhweuyehyuwyueuweydwneuyewhueyywqeuiqdjhewuidijhdf8374ur9874urkdriufjedekmfr4i9853ewrfsdikjvirfjioieru5984fkru84844844848484848jufrjrffjjfcvjjccjjjjjfjfjfghfrgvhskjvfdfndshfgkljdgjdfjtgfglgfmkjheklifrkmesfjoeiorjesjdejhyjjvodufjioruijgifrtjrfweiujjweicujejideiuduiekrujuuijjkjeeu84rie8u38ii4ro4rr498u4e9r94i9090k9e0wmdclk       345564-5645654647637=`1122343455565676788990o-0==757566y78=bhsdvbchysbvuivhdhfubjhdvhnyhjnjhdhfcvjhkdbchbjdfsdbhfdfeklrfhekjflybfherjhynfkldehrfejhrfhehugyurhkjiettrygjukheryugrejufiygfiefgrhvvjkbgnjhbcverghbbjekrgbhrgbrgbffgrgguujhrgjbgurbgrggeggrfejkdlkuifdjoisdbfkjfdfhokjsdmcsdjnmjkchujmdidsujhcsdajudiojasdjiklo/adijokoskdsdikskidosjikdossdjijkdsdsddjlkdiddmkowepiwelikodwidieidlwekekdm,kdwkwlieejs.lw.ke,sekevmqqwv 
woeo

Reply:
Ok, you have a point. I am already bending my definition of error.

Reply:
Na, I think it is good enough for now :-) I finally want to carry on with availability distribution.